app/helpers/admin/site_maps_helper.rb

```diff
@@ -3,12 +3,26 @@
 module Admin::SiteMapsHelper
   include Admin::AdminHelper
 
+  def sitemap_pages
+    @sitemap_pages ||= Page.where(:_type.in => %w(Page SetPage)).entries
```

Comment:
Well all pages subclass `Page` including set entries – which we don't want to show up in the sitemap. Not sure of a better way.